test/integration/index.html

```diff
@@ -0,0 +1,34 @@
+<body>
+    <script src="../../node_modules/scratch-vm/dist/web/scratch-vm.js"></script>
+    <script src="../../node_modules/scratch-storage/dist/web/scratch-storage.js"></script>
+    <script src="../../dist/web/scratch-render.js"></script>
+    <canvas id="test" width="480" height="360"></canvas>
+    <input type="file" id="file" name="file">
```

Comment:
This input doesn't have a closing tag

Reply:
`<input>` is a "void element" and requires no closing tag (in fact adding one can be bad) -- I could do the `<input />` thing but its also not nessecary

http://www.456bereastreet.com/archive/201005/void_empty_elements_and_self-closing_start_tags_in_html/